pkg/endpoint/endpoint.go

```diff
@@ -280,17 +284,18 @@ func (p *Policy) ValidateCertificateRequest(request *certificate.Request) error
 			return fmt.Errorf(organizationUnitError, parsedCSR.Subject.OrganizationalUnit, p.SubjectOURegexes)
 		}
 
-		if !isComponentValid(parsedCSR.Subject.Country, p.SubjectCRegexes, false) {
-			return fmt.Errorf(countryError, parsedCSR.Subject.Country, p.SubjectCRegexes)
+		if !isComponentValid(parsedCSR.Subject.Province, p.SubjectSTRegexes, false) {
```

Comment:
I changed the order, so it matches the ordering that is used below. This should not have any noteworthy effects on the behavior of this function.